Assets/Scripts/switchScene.cs

```diff
@@ -9,10 +9,18 @@ public class switchScene : Interact {
 
     public override void interact() {
 		if (isOpen) {
-			SceneManager.LoadScene (nextScene);
+			StartCoroutine ("changeNextScene");
 		}
     }
 
+	IEnumerator changeNextScene(){
+		MechanicAudioManager.getInstance ().playUnlockDoorSound ();
+
+		yield return new WaitForSeconds(MechanicAudioManager.getInstance().unlockDoorClip.length*0.5f);
+
+		SceneManager.LoadScene (nextScene);
```

Comment:
What's the *0.5f for? Why are we not waiting for the full unlock door clip time?